### PR TITLE
Make time && stress tests independent from IEDeveloperPackage

### PR DESCRIPTION
**Pull request by ishariko**
### Details:
 - Get rid of InferenceEngineDeveloperPackage_DIR in order to provide possibility to build tests on installed OpenVINO. To achieve it, include gflags and gtest building into stress tests build

### Tickets:
 - 49280

### CODE REVIEW

**Comment by ishariko** [comment]
This MR need merge together with CI [PR](https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/230).

**Review by just-sparta — APPROVED** [review]

File: tests/stress_tests/README.md [hunk 1]
```diff
@@ -32,27 +32,12 @@ one at a time to mitigate memory statistics pollution. You can use
...
+Before build the tests, open a terminal, set OpenVINO™ environment, and after that
+run the commands below:
 ``` bash
-(
-export OPENVINO_BUILD_DIR=$(git rev-parse --show-toplevel)/build
-mkdir -p build && cd build && \
-cmake -DInferenceEngineDeveloperPackage_DIR=$OPENVINO_BUILD_DIR .. && make -j$(nproc) \
-)
+mkdir build && cd build
+cmake .. && make
```
Comment (azhogov) [inline tests/stress_tests/README.md:40]: make -j$(nproc)

File: tests/stress_tests/README.md [hunk 2]
```diff
@@ -32,27 +32,12 @@ one at a time to mitigate memory statistics pollution. You can use
...
-Stress tests depend from the Inference Engine Developer Package located in the
-`openvino` build directory.
-
-In the command line snippet bellow, it is assumed that the Inference Engine
-Developer Package CMake module can be found in the directory `build` under
-`openvino` repository root.
-
+To build the tests, you need to have OpenVINO™ installed or build from source.
+Before build the tests, open a terminal, set OpenVINO™ environment, and after that
+run the commands below:
```
Comment (azhogov) [inline tests/stress_tests/README.md:37]: add setupvars

File: tests/time_tests/README.md [hunk 3]
```diff
@@ -19,17 +19,6 @@ mkdir build && cd build
 cmake .. && make time_tests
 ```
```
Comment (azhogov) [inline tests/time_tests/README.md:21]: setupvars?

File: tests/stress_tests/CMakeLists.txt [hunk 4]
```diff
@@ -12,7 +19,8 @@ endif()
 # Define directory where artifacts will be placed
 set(OUTPUT_ROOT ${CMAKE_CURRENT_SOURCE_DIR})
 
-find_package(InferenceEngineDeveloperPackage REQUIRED)
+# Search OpenVINO Inference Engine installed
+find_package(InferenceEngine REQUIRED)
```
Comment (ilyachur) [inline tests/stress_tests/CMakeLists.txt:23]: Maybe can we  start to use new OpenVINO cmake config?

**Review by ilyachur — APPROVED** [review]

File: tests/stress_tests/README.md [hunk 5]
```diff
@@ -32,27 +32,13 @@ one at a time to mitigate memory statistics pollution. You can use
...
 ``` bash
-(
-export OPENVINO_BUILD_DIR=$(git rev-parse --show-toplevel)/build
-mkdir -p build && cd build && \
-cmake -DInferenceEngineDeveloperPackage_DIR=$OPENVINO_BUILD_DIR .. && make -j$(nproc) \
-)
+source <OpenVINO_install_dir>/bin/setupvars.sh
```
Comment (ilya-lavrenov) [inline tests/stress_tests/README.md:39]: `bin/setupvars.sh` -> `setupvars.sh`

**Comment by azhogov** [comment]
CI passed: https://github.com/intel-innersource/frameworks.ai.openvino.ci.product-configs/pull/230

**Pull request merged**